roles/keystone/tasks/main.yml

```diff
@@ -211,3 +211,12 @@
   tags:
     - serverspec
   when: serverspec.enabled|default('False')|bool
+
+- name: setup keystone inspec checks
+  blockinfile:
+    dest: "{{ inspec.control_path }}"
```

Comment:
I wish we could do this too, but earlier I got an error when using require_controls on the same item (inspec-openstack-security) in two different files:
```
/opt/inspec/embedded/lib/ruby/gems/2.3.0/gems/inspec-1.7.1/lib/inspec/dsl.rb:62:in `block in filter_included_controls': undefined method `+' for nil:NilClass (NoMethodError)
```
Maybe this is a nice feature that could be pushed to inspec?